internal/logger.go

```diff
@@ -2,25 +2,28 @@ package internal
 
 import (
 	"fmt"
+	"log"
+	"os"
 	"time"
 )
 
 // Logger represents a logger.
 type Logger struct {
 	currentLevel VerbosityLevel
+	logger       log.Logger
 }
 
 // NewLogger creates a new instance of a logger.
 func NewLogger(level VerbosityLevel) *Logger {
-	return &Logger{currentLevel: level}
+	return &Logger{currentLevel: level, logger: *log.New(os.Stderr, "", 0)}
```

Comment:
No point mixing `log.Logger` with `fmt`. You could either use `fmt.Fprint(os.Stderr,...)` or use `log.New(os.Stderr, "", log.LstdFlags)`.